tests/installation/enable_y2_firstboot.pm

```diff
@@ -16,11 +16,39 @@ use strict;
 use warnings;
 use testapi;
 use utils qw(zypper_call clear_console);
+use scheduler 'get_test_suite_data';
+use XML::LibXML;
+
+sub edit_control_file {
+    my $control_file = shift;
+    # my $xml_data = script_output("cat $control_file");
+
+    open(my $fh, '>', 'xml_data.xml');
+    print $fh script_output("cat $control_file");
+    close $fh;
+
+    my $parser = XML::LibXML->new();
+    my $doc    = $parser->parse_file('xml_data.xml');
+
+    my $xpc = XML::LibXML::XPathContext->new($doc);
+    $xpc->registerNs('ns', 'http://www.suse.com/1.0/yast2ns');
+
+    my ($node_to_edit) = $xpc->findnodes("/ns:productDefines/ns:workflows/ns:workflow/ns:modules/ns:module[./ns:label[text()='Host Name']]/ns:enabled");
+    $node_to_edit->removeChildNodes();
+    $node_to_edit->appendText('true');
+    $node_to_edit->to_literal;
+    my $new_control_file = $doc->toString;
+    $new_control_file =~ s/\"/\\\"/g;
+    assert_script_run("echo \"$new_control_file\" > $control_file");
```

Comment:
Instead of that echo, it would be preferable to find a way to upload the modified xml.

Reply:
you can always upload it to the OpenQA WebUI with http://open.qa/api/testapi/#_upload_asset for purpose of debugging or providing logs.